guides/release/routing/loading-and-error-substates.md

```diff
@@ -106,21 +106,20 @@ don't immediately resolve, a [`loading`](https://www.emberjs.com/api/ember/relea
 
 ```javascript {data-filename=app/routes/foo-slow-model.js}
 import Route from '@ember/routing/route';
+import { action } from '@ember/object';
 
-export default Route.extend({
+export default class FooSlowModelRoute extends Route {
```

Comment:
Can this avoid using `foo`?

Reply:
The use of `Foo` is all through these examples. I will open an issue to change it. `Foo Bar` examples are not really great when we have so many real-life use cases available to us.

Reply:
https://github.com/ember-learn/guides-source/issues/646